src/pip/_internal/resolution/legacy/resolver.py

```diff
@@ -146,8 +152,9 @@ def __init__(
         self.use_user_site = use_user_site
         self._make_install_req = make_install_req
 
-        self._discovered_dependencies = \
-            defaultdict(list)  # type: DiscoveredDependencies
+        self._discovered_dependencies = defaultdict(
+            list
+        )  # type: DiscoveredDependencies
```

Comment:
Ugh.

Reply:
I blame Python 2 (type comments) and over-long variable names for that.